### PR TITLE
feat: guest login added

### PR DESCRIPTION
**Pull request by Aazam10**
1.Added login for guest user.
You can input user credentials
 email: "adarshbalika@gmail.com",
  password: "adarshBalika123"
 or use guest btn login.

Signup functionality has not been added yet will add in the end just added login functionality to access private routes of api

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,12 +1,23 @@
 import { FaMoon, FaSun } from "react-icons/fa";
 import "./Navbar.css";
+import { useAuth } from "../../context";
+import { Link } from "react-router-dom";
 const Navbar = () => {
+  const { authState, authDispatch } = useAuth();
+  const { token } = authState;
+  console.log(token);
```
Comment (kart-c) [inline src/components/Navbar/Navbar.jsx:8]: remove the console logs

Comment (AnkurChunekar) [inline src/components/Navbar/Navbar.jsx:8]: remove unwanted console logs

**Review by kart-c — APPROVED** [review]

**Review by AnkurChunekar — APPROVED** [review]
- Rest everything looks good to merge.
- Functionality works as specified.
- Great work @Aazam10

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Aazam10) [reply src/components/Navbar/Navbar.jsx:8]: yes will remove thanks for pointing out

**Pull request merged**